data/json/items/gun/shot.json

```diff
@@ -1,5 +1,29 @@
 [
   {
+    "id": "abzats",
+    "copy-from": "shotgun_base",
```

Comment:
This should probably actually `copy-from` the m2 browning as specified in the `description`